frontend/package.json

```diff
@@ -36,6 +36,7 @@
     "ember-cli-surge": "^1.1.20",
     "ember-cli-template-lint": "^0.7.3",
     "ember-cli-uglify": "^1.2.0",
+    "ember-cli-underscore": "0.0.3",
```

Comment:
okay, no issue.
 What about the selector problem? If it because the plot was created in a  component? Do I have to place the replot method also in the same component and reference it somehow? thanks